### DISCUSSION TITLE
La KATA 2 sobre el ambiente virtual como se va entregar

### DISCUSSION THREAD

**Discussion by RaulGuereca** [LaunchX-InnovaccionVirtual/CursoIntroPython#279] [Q&A] [answered]
Como se va entregar la KATA2 sobre el ambiente virtual si solo estamos trabajando directamente con instrucciones en la consola. Como va ser el entregable en este ejercicio.

**Comment by enery** [comment, marked as answer]
hola, tómale una captura a la consola y eso enviaras en esa kata

**Reply by RaulGuereca** [reply]
muchas gracias ...
